app/assets/stylesheets/application/member-card.scss

```diff
@@ -1,16 +1,26 @@
+.column--cards {
+  display: flex;
+  flex-direction: column;
+}
+
 .to-member-card {
   & {
     margin: 16px 0;
     width: 100%;
     max-width: 600px;
-    height: 178px;
+    min-height: 178px;
     background-color: white;
     box-shadow: 0 0 3px $palette-grey;
-    display: inline-block;
+    border-radius: 0.3rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    box-sizing: border-box;
```

Comment:
I can see the value of these 3 `display` `flex-direction` and `justify-content` so the card footer is always at the bottom but is `box-sizing` giving any value here?